main.py

```diff
@@ -10,11 +10,16 @@
 from impf.alert import send_alert
 from impf.api import API
 from impf.browser import Browser
+from impf.exceptions import WorkflowException
 
 logger = logging.getLogger(__name__)
 b = None  # helper variable for keeping browser open
 
 
+class WorkflowException(Exception):
+    pass
+
+
```

Comment:
Typo? Got it here and in exceptions

Reply:
Thanks, I missed that while refactoring my WIP. Fixed